modules/memberships/class-jetpack-memberships.php

```diff
@@ -35,6 +35,21 @@ class Jetpack_Memberships {
 	 * @var string
 	 */
 	private static $button_block_name = 'membership-button';
+
+	/**
+	 * These are defaults for wp_kses ran on the membership button.
+	 *
+	 * @var array
+	 */
+	private static $tags_allowed_in_the_button = array(
+		'br'     => array(),
+		'em'     => array(),
+		'strong' => array(),
+		'p'      => array(),
+		'img'    => array(
```

Comment:
Do we want to allow images and p elements inside a button? That seems a bit odd to me, that's not the kind of markup I would expect there. Would that be okay accessibility-wise as well?

Reply:
I didn't expect it there as well - that's why the linebreaks don't work :)

But when you think about it, - I am 100% sure someone will try to design an image and put it inside the block.
Just have a look at the nice block @rantoncuadrado  has put together:

<img width="551" alt="remote-training-fundamentals-for-companies -10-tips-2019-05-15-16-02-16" src="https://user-images.githubusercontent.com/3775068/58040728-6e455a00-7b36-11e9-9008-f24eefa92b33.png">

Of course, it was possible thanks to extensive knowledge of CSS, but some of our customer will try to do this using other means

I can use `wp_kses_post` as Mailchimp PR you linked - would that be ok?

Reply:
While that looks great, I'm not sure that's something we'd want to encourage. Semantically I think it would make sense for a site owner to implement something like that using multiple blocks. I could imagine relying on column blocks, and creating a reusable block containing 2 columns, an image, some text, and our Membership button:

![image](https://user-images.githubusercontent.com/426388/58041969-b87c0a80-7b39-11e9-8797-8752c5ad1557.png)

I think it would make more sense semantically, wouldn't you think?

Reply:
👍 
Changed to only allow `<br/>`, as in #11223.  We can always change it later to allow more if we have a public outcry :)

Reply:
> I could imagine relying on column blocks, and creating a reusable block containing 2 columns, an image, some text, and our Membership button

By the way, it'd be great to offer these as templates :)

Reply:
cc @iamtakashi